.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     environment: BUF
     needs: run_checker
-    if: ${{ needs.run_checker.outputs.run_release_proto }} && github.repository_owner == 'astriaorg'
+    if: ${{ needs.run_checker.outputs.run_release_proto && github.repository_owner == 'astriaorg' }}
```

Comment:
Wow, that's a good catch. I don't quite get the semantics of this. Does `&&` not work outside of `${{ }}`?

Reply:
This was actually not the issue, but updates fix.